README.md

```diff
@@ -82,6 +82,12 @@ No installation needed
 - GitHub: [@githubhandle](https://github.com/firepicaso)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/mustakim-masum/)
 
+👤 **Ian Kiema**
+
+- GitHub: [@githubhandle](https://github.com/iankiema)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
```

Comment:
- [ ] The link provided for the twitter handle is broken as the account doesn't exist, please kindly fix this broken link in your readme by providing the appropriate link to your twitter account :+1: :sparkles: 
- [ ] Also the link to your linkedIn account should also be provided for correctly :+1: